gqlgen.yml

```diff
@@ -27,6 +27,8 @@ models:
     model: github.com/evergreen-ci/evergreen/rest/model.APIPatch
   Parameter:
     model: github.com/evergreen-ci/evergreen/rest/model.APIParameter
+  ParameterInput:
```

Comment:
Parameter is already defined above this; is there a reason that can't be used? I don't see anything else here that is defined twice.

Reply:
yeah, it's annoying. It has to be defined again.

Reply:
Could you explain this? Is it just convention of input things being called input? Can the existing "Parameter" be renamed?

Reply:
discussing over slack